control-plane/agents/src/bin/ha/node/server.rs

```diff
@@ -124,34 +124,41 @@ impl NodeAgentSvc {
         // Open connection to the new nexus: ANA will automatically create
         // the second path and add it as an alternative for the first broken one,
         // which immediately resumes all stalled I/O
-        let mut subsystem = match client
-            .nvme_connect(NvmeConnectRequest {
-                uri: new_path.clone(),
-                publish_context: publish_context.map(|map| MapWrapper { map }),
-            })
-            .await
-        {
-            Ok(_) => match get_subsystem(&parsed_uri) {
-                Ok(subsystem) => {
-                    tracing::info!(new_path, "Successfully connected to NVMe target");
-                    subsystem
+        let mut subsystem = match get_subsystem(&parsed_uri) {
+            Ok(subsystem) => {
+                tracing::info!("Reusing existing subsystem path");
```

Comment:
Can we include subsystem details in the trace?

Reply:
This, bit of code will be changed, as now we fetch in nvmfAttach.